src/renderer/services/docker/chain.ts

```diff
@@ -56,6 +65,23 @@ export class BeaconChain extends Container {
         return `${network}-beacon-node`;
     }
 
+    public static getNetworkFromContainerName(containerName: string): SupportedNetworks|undefined {
+        const name = containerName.split("-")[0];
+        if (name === SupportedNetworks.PRYSM) {
+            return SupportedNetworks.PRYSM;
+        }
+    }
+
+    private static async loadStoppedChain(name: string, image: string): Promise<void> {
+        const bc = new BeaconChain({
+            name,
+            image,
+        });
+        await bc.startStoppedContainer();
+        const network = BeaconChain.getNetworkFromContainerName(name);
```

Comment:
Maybe it would be better so save those connections in database?

Reply:
We have validator - network connection. But this is just docker image / beacon chain. I am not sure if we need this connection in any other way so didn't want to overcomplicate. We have a clear naming right now that we use it the network in container name.